src/actions/clientActions.js

```diff
@@ -1,22 +1,35 @@
 import { CLIENT_CHANGE } from '../constants/actionTypes'
+import { SMALL_SCREEN, SMALL_SCREEN_BREAKPOINT, MEDIUM_SCREEN, MEDIUM_SCREEN_BREAKPOINT, LARGE_SCREEN } from '../constants/screenSizes'
```

Comment:
I think it would be better to import this as a single object and refer to it.

`import * as screenSizes from '...'`